ShapeVariationAnalyzer/ShapeVariationAnalyzer.py

```diff
@@ -2246,8 +2246,23 @@ def addColorMap(self, table, dictVTKFiles):
                     polyData = append.GetOutput()
 
                     vtkFile = os.path.splitext(vtkFile)[0]+'.vtk'
+                elif os.path.basename(vtkFile).endswith(".srep.json"):
+                    storage = slicer.mrmlScene.AddNewNodeByClass('vtkMRMLSRepStorageNode')
+                    storage.SetFileName(vtkFile)
```

Comment:
Keeping the internal variable name as `vtkFile` is confusing as it is a `.srep.json`. I suggest to introduce an intermediate variable:

```
srepFile = vtkFile
```

Reply:
Updated to use a generic `filename` variable when we don't know what type it is. `vtkFile` is set by each branch of the `if` with an actual `.vtk` filename.